src/mpiwrapper.f

```diff
@@ -5,17 +5,28 @@ subroutine mpiwrapper_export_fortran_constants
       include "mpiabif.h"
 
 !     Assert that MPI_STATUS_SIZE <= MPIABI_STATUS_SIZE
-      integer(1 - max(0, MPI_STATUS_SIZE - MPIABI_STATUS_SIZE)) check_mpi_status_size
+      integer(1 - max(0, MPI_STATUS_SIZE - MPIABI_STATUS_SIZE))
+     &     check_mpi_status_size
 
-!     integer(1 - abs(MPI_ADDRESS_KIND - 4)) check_mpi_address_kind
+!     integer(1 - abs(MPI_ADDRESS_KIND - 8)) check_mpi_address_kind
       integer(1 - abs(MPI_COUNT_KIND - 8)) check_mpi_count_kind
       integer(1 - abs(MPI_INTEGER_KIND - 4)) check_mpi_integer_kind
       integer(1 - abs(MPI_OFFSET_KIND - 8)) check_mpi_offset_kind
 
       include "mpiabi_defn_constants_fortran.h"
 
+      print '("MPIwrapper: MPI_IN_PLACE loc: ", i0)', loc(MPI_IN_PLACE)
+      print '("MPIwrapper: MPI_IN_PLACE: ", i0)', MPI_IN_PLACE
+
```

Comment:
These print statement should be removed or guarded, they caused an issue with the `configure` step of ScaLAPACK

Reply:
Removed. Thanks!